include/quic/gnutls_crypto.hpp

```diff
@@ -188,47 +309,58 @@ namespace oxen::quic
         // Construct from raw Ed25519 keys
         GNUTLSCreds(std::string ed_seed, std::string ed_pubkey);
 
+      protected:
```

Comment:
superfluous (probably there was something in here before that got moved?)

Reply:
just protecting my privates ya know